### ISSUE TITLE
Mac: After uploading a file with the keyboard shortcut CTRL/CMD+U, put the focus in the text box

### ISSUE THREAD

**Issue by jasonblais** [mattermost/desktop#341]
I confirm (by marking "x" in the [ ] below): 
- [X] This is not a feature idea. Feature ideas go here: http://www.mattermost.org/feature-ideas/
- [X] This is not a troubleshooting question. Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/
- [X] This is a product defect issue that can be reproduced using the steps listed below.
- [X] I have read [CONTRIBUTING.md](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md). 

---
#### Summary

Mac: After uploading a file with the keyboard shortcut, put the focus in the text box
#### Steps to reproduce
- Operating System: Mac
- Mattermost Desktop App v3.4.1
- Mattermost Server v3.4
1. Upload file using CMD or CTRL + U
2. Choose file with keyboard + hit enter to select
3. Hit enter to send the message
#### Expected behavior

Describe your issue in detail.
#### Observed behavior

Cursor should be in message input box after selecting the file

**Comment by yuya-oc** [comment]
Note:

It seems that [the server always puts the focus in the text box](https://github.com/mattermost/platform/blob/9f6ba5e031ee4a621a319523823562916e6532c2/webapp/components/create_post.jsx#L215), and the problem is not reproduced on Windows.

I also tried a very simple electron app without webview. It doesn't reproduce the problem.
```javascript
const {app, BrowserWindow} = require('electron');

var mainWindow;
app.on('ready', () => {
  mainWindow = new BrowserWindow();
  mainWindow.loadURL(MATTERMOST_SERVER_URL);
});
```

So there might be something wrong in our implementation. Possibly webview's focus?

**Comment by jasonblais** [comment]
Requires further investigation so removing milestone for now

**Comment by yuya-oc** [comment]
The point is that the dialog is attached to the main window on macOS. It prevents our event handlers, so the webview couldn't get focus correctly.

To solve this, an event which is not supported by Electron is necessary. I confirmed the problem was solved by using modified Electron. So I'll prepare a PR for this.

But probably it doesn't meet the next milestone even if the PR is merged (it would be still beta when we release the next one).

**Comment by yuya-oc** [comment]
Necessary event was added to Electron. This issue can be fixed in future.

https://github.com/electron/electron/pull/9108

**Comment by jasonblais** [comment]
@yuya-oc How much work would you say it would be to fix this issue? We got another report about it, where you can hit "Enter" to send a file attachment, since the focus is away from the message input box.

**Comment by yuya-oc** [comment]
Almost done in another branch. We need to wait Electron 1.6.8 which is still beta.

https://circleci.com/gh/yuya-oc/desktop/239#artifacts

**Comment by yuya-oc** [comment]
Closed via #525
